Source/Data/Controls.cs

```diff
@@ -12,11 +12,13 @@ public static class Controls
 	public static readonly VirtualButton Dash = new("Dash", .1f);
 	public static readonly VirtualButton Climb = new("Climb");
 	public static readonly VirtualButton Confirm = new("Confirm");
+	public static readonly VirtualButton ControllerConfirm = new("ControllerConfirm");
```

Comment:
Oh, that's a really good find. I didn't even see this

Reply:
I added that button because when you are typing in the InputField, if you press C on the keyboard (default confirm key) it will show the OnScreenKeyboard menu. Where if you have another bind for the controller, you only need to check that and not the keyboard, allowing you to still press the C and X keys.